packages/server/tests/clients/servicesClient.ts

```diff
@@ -0,0 +1,246 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+    AccountBalanceQuery,
+    AccountId,
+    AccountInfoQuery,
+    Client,
+    ContractCreateTransaction,
+    ContractExecuteTransaction,
+    ContractFunctionParameters,
+    FileCreateTransaction,
+    Hbar,
+    HbarUnit,
+    Key,
+    PrivateKey,
+    Query,
+    TokenAssociateTransaction,
+    TokenCreateTransaction,
+    Transaction,
+    TransactionResponse,
+    TransferTransaction
+} from '@hashgraph/sdk';
+import { Logger } from 'pino';
+import { Utils } from '../helpers/utils';
+import { ethers } from 'ethers';
+
+const supportedEnvs = ['previewnet', 'testnet', 'mainnet'];
+
+export default class ServicesClient {
+
+    private readonly DEFAULT_KEY = new Key();
+    private readonly logger: Logger;
+    private readonly network: string;
+
+    public readonly client: Client;
+
+    constructor(network: string, accountId: string, key: string, logger: Logger) {
+        this.logger = logger;
+        this.network = network;
+
+        if (!network) network = '{}';
+        const opPrivateKey = PrivateKey.fromString(key);
+        if (network.toLowerCase() in supportedEnvs) {
```

Comment:
Verify that this works cause i'm not sure it does.
In my other PR i had to change it to 
```suggestion
        if (supportedEnvs.includes(network.toLowerCase())) {
```